gen2-color-isp-raw/main.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import cv2
+import numpy as np
+import depthai as dai
+
+streams = []
+# Enable one or both streams
+streams.append('isp')
+#streams.append('raw') # TODO: implement unpacking
+
+display_scale = 0.3  # configurable
+
+print("depthai version:", dai.__version__)
+pipeline = dai.Pipeline()
+
+cam = pipeline.createColorCamera()
+cam.setEnablePreviewStillVideoStreams(False)
```

Comment:
Can `raw`, `isp` and other outputs be used concurrently?
Or is this a limitation?
If they can be used, then we can rather do the check for disabling intermediate processing on device side by checking if the output is connected somewhere

Reply:
Yes, the outputs can be used together with the other post-processed outputs. Disabled them here to reduce the loading on device (and also to avoid the possibility of non-working with some non-standard higher resolutions - around 5K / IMX283).
Definitely we should check inside the node if the outputs are connected, and enable the post-processor accordingly. I just didn't find the API.

Reply:
Okay great - then lets put this as an issue to depthai-device (or core) repository, and we'll pick it up later